app-modules/task/tests/TaskAssignmentTest.php

```diff
@@ -34,49 +34,42 @@
 </COPYRIGHT>
 */
 
-use AdvisingApp\Authorization\Models\Permission;
+use AdvisingApp\Authorization\Enums\LicenseType;
+use AdvisingApp\Task\Filament\Resources\TaskResource;
 use AdvisingApp\Task\Models\Task;
 use AdvisingApp\Task\Notifications\TaskAssignedToUserNotification;
 use App\Models\User;
 use Illuminate\Support\Facades\Notification;
 
+use function Pest\Laravel\actingAs;
+
 beforeEach(function () {
     Notification::fake();
 });
 
-it('creates the proper permissions record when a Task is created', function () {
-    $task = Task::factory()->create();
-
-    expect(Permission::where('name', "task.{$task->getKey()}.update")->exists())->toBeTrue();
-});
-
-it('gives the proper permission to the creator of a Task', function () {
-    /** @var Task $task */
-    $task = Task::factory()->create();
-
-    expect($task->createdBy->can("task.{$task->getKey()}.update"))->toBeTrue();
-});
-
-it('gives the proper permission to the assigned User of a Task on create and update', function () {
+it('gives the proper permission to the assigned User of a Task on update', function () {
```

Comment:
I think we can just remove this test as well. Task Assignment now does not create or do anything to the access of the User directly.

Access is intrinsic based on the assignment. Which can be tested in the Edit page test.